taxi/views.py

```diff
@@ -85,5 +88,39 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
-    model = Driver
+    model = get_user_model()
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
good job, but also you need to change queryset, since it's overriding model. And if you change queryset, you won't longer need model.

Reply:
fix that